src/templates/community.js

```diff
@@ -35,7 +35,9 @@ export const pageQuery = graphql`
         prev
       }
       fields {
-        path
+        path {
+          id
```

Comment:
@KyleAMathews any idea why this now requires `{ ... }`?

<img width="547" alt="screen shot 2018-07-16 at 9 23 53 pm" src="https://user-images.githubusercontent.com/1571667/42796960-c48c5c66-8941-11e8-908d-d3de38a422b4.png">